exporter/awsemfexporter/metric_translator.go

```diff
@@ -118,6 +134,268 @@ func (dps DoubleHistogramDataPointSlice) At(i int) DataPoint {
 	return dps.DoubleHistogramDataPointSlice.At(i)
 }
 
+
+// Retrieve namespace for given set of metrics from user config 
+func getNamespace(rm *pdata.ResourceMetrics, namespace string) (string) {
+	if len(namespace) == 0 && !rm.Resource().IsNil() {
```

Comment:
Why was `!rm.Resource().IsNil()` added? IIRC, this is a deprecated function. Please build this project again to confirm.